doc/dev/index.rst

```diff
@@ -1021,6 +1050,53 @@ drive::
 
     $ ceph-workbench ceph-qa-suite --suite dummy
 
+Your first run of `ceph-workbench ceph-qa-suite`_ on a given tenant will
+take a long time to complete because it downloads a VM image. The images
+are cached in FIXME, so they are only downloaded once.
+
+Although ``dummy`` suite does not run any tests, in all other respects it
+behaves just like a teuthology suite and produces some of the same
+artifacts (see `Artifacts produced by teuthology-suite`_).
+
+You can tell whether the ``dummy`` suite completed successfully by FIXME.
+
+The last line of output looks something like this::
+
+    ssh -i /home/smithfarm/.ceph-workbench/teuthology-myself.pem ubuntu@149.202.168.201
+
+With this command you can access the teuthology node that was just spun
+up. Since the teuthology node is persistent, it may be useful to save this
+command for later reference - for example, by making it an alias::
+
+    alias teuthology="ssh -i . . ."
+
+Run a standalone test
+---------------------
+
+The standalone test explained in `Reading a standalone test`_ can be run
+with the following command::
+
+    $ ceph-workbench ceph-qa-suite --suite rados/singleton/all/admin-socket.yaml
+
+This will run the suite shown on the current ``master`` branch of
+``ceph/ceph.git``. You can specify a different branch with the ``--ceph``
+option, and even a different git repo with the ``FIXME`` option. (Run
+``teuthology-suite --help`` on the teuthology node for an up-to-date list
+of available options.)
+
```

Comment:
the above with the suggested fixes look good and can be merged. the following lines are WIP I suppose ;-)